web/src/components/dashboard/OpportunityCard.test.tsx

```diff
@@ -85,31 +85,27 @@ describe("OpportunityCard", () => {
     expect(screen.getByText("Rolling Application")).toBeInTheDocument();
   });
 
-  it("displays full first 150 characters of description", () => {
-    const characters = Array(151).fill("a").join("");
-    const expectedTextOnPage = `${Array(150).fill("a").join("")}...`;
+  it("displays OPPORTUNITY_CARD_MAX_BODY_CHARS (minus 3) chars & ellipsis in description when sidebarCardBodyText is OPPORTUNITY_CARD_MAX_BODY_CHARS + 1 length", () => {
+    const characters = Array(OPPORTUNITY_CARD_MAX_BODY_CHARS + 1)
+      .fill("a")
+      .join("");
+    const expectedTextOnPage = `${Array(OPPORTUNITY_CARD_MAX_BODY_CHARS - 3)
+      .fill("a")
+      .join("")}...`;
 
-    const opportunity = generateOpportunity({ descriptionMd: characters });
+    const opportunity = generateOpportunity({ sidebarCardBodyText: characters });
     render(<OpportunityCard opportunity={opportunity} urlPath="funding" />);
     expect(screen.getByText(expectedTextOnPage)).toBeInTheDocument();
   });
 
-  it("truncates markdown without showing characters on page for bold", () => {
-    const characters = Array(145).fill("a").join("");
-    const boldContent = `${characters} *a bold text*`;
+  it("displays all characters of description", () => {
+    const characters = Array(OPPORTUNITY_CARD_MAX_BODY_CHARS - 10)
```

Comment:
[sand] I would also add a case for when the description is exactly `OPPORTUNITY_CARD_MAX_BODY_CHARS` characters long.

Reply:
[pebble] Are we going to add a test that is is exactly `OPPORTUNITY_CARD_MAX_BODY_CHARS` as JP suggested? If so,  I don't think it's a heavy lift and think we can do it now.

Reply:
[dust/pebble] for readability sake, I think it might be worth restructuring these tests, maybe something along the lines of this: 
```
describe("content truncation based on OPPORTUNITY_CARD_MAX_BODY_CHARS"){
  it("displays ellipsis and truncated content when character count is over the max",  () => {...})
  it("displays the entire content when character count is under the max",  () => {...})
  it("displays the entire content when character count is exactly the max",  () => {...})
}
```
You can rename the describe block as you see fit, but I think grouping them makes it more readable. @AnJuHyppolite @jphechter what do y'all think?

Reply:
I like this. We may not consistently do it across the codebase, but it does make the tests more readable. `+1` on implementing